framework/src/modules/chain/transport.js

```diff
@@ -77,7 +77,7 @@ class Transport {
 			scope.config.nonce,
 			scope.config.broadcasts,
 			scope.config.forging.force,
-			scope.logic.transactionPool,
+			scope.modules.transactionPool,
```

Comment:
Why do we bind transactionPool in the onBind function this file already receives it in the constructor?

Reply:
This is because I didn't want to touch the bind method (since it will be removed), and it will break the test ='(
forgers, transport and loader needs to be refactored after the block PR